ntbs-service/Pages/Shared/_NotificationLayoutWithSideMenu.cshtml

```diff
@@ -17,33 +17,34 @@
 }
 
 <nhs-width-container container-width="Standard">
-    <form method="post" autocomplete="off">
+    <form-leave-checker inline-template>
+        <form ref="notificationForm" method="post" autocomplete="off">
```

Comment:
If form-leave-checker is only ever going to be the direct parent of the form it's checking then we don't need to use refs, but happy to be ignored - this one could fall down to stylistic preference.

Reply:
I have moved form into Layout